site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -234,27 +233,21 @@
   }
 
   {% if viewer.isGrader() %}
-    window.isAudibleAlertEnabled = false;
+    window.audible_notifications_enabled = false;
 
     try {
         window.notificationSound = new NotificationSound();
-        document.querySelector('.disable-audible').style.display = 'none';
-        document.querySelector('.enable-audible').style.display = 'inline-block';
     } catch (e) {
         console.error(e);
     }
   {% endif %}
 
   function toggleAudibleAlert() {
-      if (window.isAudibleAlertEnabled) {
-          document.querySelector('.disable-audible').style.display = 'none';
-          document.querySelector('.enable-audible').style.display = 'inline-block';
-          window.isAudibleAlertEnabled = false;
+      if (window.audible_notifications_enabled) {
+          window.audible_notifications_enabled = false;
```

Comment:
`function toggleAudibleAlert() {
    window.audible_notifications_enabled = !window.audible_notifications_enabled;
}
`
How about more concise version?

Reply:
That should be left to a future PR. The entire push/sound notification system should probably be refactored; it is fairly convoluted.